src/main/webapp/components/AddDocumentDialog.js

```diff
@@ -94,6 +105,43 @@ export const AddDocumentDialog = hh(class AddDocumentDialog extends Component {
     });
   };
 
+  validEmail = (email) => {
```

Comment:
This should be a generalized function so we can keep the regex in one spot. I can see this being copy/pasted unnecessarily.